source/source_integration_test.go

```diff
@@ -30,7 +30,17 @@ import (
 	"github.com/matryer/is"
 )
 
-func TestSource_Read_NoTable(t *testing.T) {
+const createTableQueryFmt = `
+CREATE TABLE %s (
+  id NUMBER GENERATED by default on null as IDENTITY, 
+  name VARCHAR2(30) NOT NULL, 
+  age NUMBER, 
+  is_active NUMBER(1, 0), 
+  PRIMARY KEY (id)
+)
+`
+
+func TestSource_noTable(t *testing.T) {
 	var (
 		ctx = context.Background()
```

Comment:
Since we're overwriting `ctx` almost immediately after this, maybe we can do it here, `ctx, cancel = ...`?

Reply:
Yes, you're right. 
So I've removed unnecessary context variables in tests.